### ISSUE TITLE
JSON of LSTMTimeSeriesRegressor doesn't match description

### ISSUE THREAD

**Issue by itinawi** [HDI-Project/MLPrimitives#90]
Currently the json file `mlprimitives/jsons/keras.Sequential.LSTMTimeseriesRegressor.json` doesn't match the description of being a single layer LSTM network.

A change needs to be made so the model has one LSTM, one dropout, and one dense output layers, since the use of dropout in the LSTM layer was deprecated and generates a warning if not addressed.
